outplacement_order_interpretor/views/res_partner_view.xml

```diff
@@ -12,4 +12,16 @@
         </field>
     </record>
 
+    <record id="view_outplacement_form" model="ir.ui.view">
+        <field name="name">outplacement.view.form.inherit.interpreter</field>
+        <field name="model">outplacement</field> 
+        <field name="inherit_id" ref="outplacement.outplacement_form_view" />
+        <field name="arch" type="xml">
+            <field name="employee_id" position="after">
+                <field name="interpreter_language"/>
```

Comment:
This field seems to be missing

Reply:
I'm sorry I was clearly completely out of it when I thought this would work.